domain/assets.go

```diff
@@ -0,0 +1,17 @@
+package domain
+
+import (
+	"time"
+)
+
+type UUID [16]byte
+
+type assets struct {
+	Id             UUID                   `db:"id"` //uuid package read
+	Status         string                 `db:"status"`
+	Category       string                 `db:"category"`
+	Purchaseat     time.Time              `db:"purchase_at"`
+	Purchasecost   float64                `db:"purchase_cost"`
+	Name           string                 `db:"name"`
+	Specifications map[string]interface{} `db:"specifiation"` //encoding json read
```

Comment:
whats the key name for 'Specifications' in db migrations?

change the db key accordingly